e2e/suite/channel_resolve_test.go

```diff
@@ -5,17 +5,17 @@ import (
 	"net/http"
 	"net/http/httptest"
 
-	. "github.com/onsi/ginkgo/v2"
-	. "github.com/onsi/gomega"
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
 	"github.com/rancher/system-upgrade-controller/e2e/framework"
 	upgradeapiv1 "github.com/rancher/system-upgrade-controller/pkg/apis/upgrade.cattle.io/v1"
 	upgradeplan "github.com/rancher/system-upgrade-controller/pkg/upgrade/plan"
 )
 
-var _ = Describe("Resolve channel", func() {
+var _ = ginkgo.Describe("Resolve channel", func() {
```

Comment:
our pattern in is to import the DSL into the package namespace - not to import it as ginkgo/gomega. These changes don't appear to be related to package updates, please drop these from the PR.